kt/godot-library/src/main/kotlin/godot/annotation/propertyHintAnnotations.kt

```diff
@@ -0,0 +1,265 @@
+package godot.annotation
+
+import godot.registration.Range
+
+/*
+All type checks will happen at the entry-generator at the moment.
+We should move that to the idea plugin at some point in time to provide IDE help and not just compilation errors.
+
+The following annotations are implemented based on https://github.com/godotengine/godot/blob/3.2/core/object.h
+ */
+
+/**
+ * Can only be used on Int properties!
+ */
+@Target(AnnotationTarget.PROPERTY)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class IntRange(val start: Int, val end: Int, val step: Int = -1, val or: Range = Range.NONE)
```

Comment:
We have a Float and Double range. Shouldn't we have LongRange then ? We only got a IntRange.

Reply:
Hmm i guess you're right.
I'll add it